### PR TITLE
sorted project updates by date on old design

### PR DESCRIPTION
**Pull request by blai0264**
# [Sort updates on project pages by date with most recent on the left (old design)](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=224225)

Sorted project updates by date

## Test Instructions

1. yarn dev
2. navigate to each project page
3. ensure dates are sorted properly

### CODE REVIEW

**Comment by andr0272** [comment]
Check out a preview of your pull request here: https://app-servicecanadalabs-dev-pr-preview-3.azurewebsites.net

File: pages/projects/benefits-navigator/index.js [hunk 1]
```diff
@@ -11,10 +11,12 @@ import { Collapse } from "../../../components/molecules/Collapse";
 import Image from "next/image";
 import stageDictionary from "../../../lib/utils/stageDictionary";
 import TextRender from "../../../components/text_node_renderer/TextRender";
+import { sortUpdatesByDate } from "../../../lib/utils/sortUpdatesByDate";
 
 export default function BenefitsNavigatorOverview(props) {
   const [pageData] = useState(props.pageData.item);
   const [updatesData] = useState(props.updatesData);
```
Comment (will0684) [inline pages/projects/benefits-navigator/index.js:18]: Let's replace the useState here with the way it's implemented on the other project pages. The way we bring in a lot of the props across the site needs to be made more consistent in general but we'll get to the rest of it later.

**Review by will0684 — CHANGES_REQUESTED** [review]

File: pages/projects/benefits-navigator/index.js (hunk 1 above)
Comment (blai0264) [reply pages/projects/benefits-navigator/index.js]: sounds good! 👍🏾

**Comment by andr0272** [comment]
Check out a preview of your pull request here: https://app-servicecanadalabs-dev-pr-preview-3.azurewebsites.net

**Review by will0684 — APPROVED** [review]
Looks good thank you!

**Pull request merged**